src/transformers/models/t5gemma/modeling_t5gemma.py

```diff
@@ -505,81 +505,6 @@ def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
         return logits
 
 
-class T5GemmaAttention(nn.Module):
```

Comment:
`_can_record_outputs` below was incorrect, which made modular create this class that isn't even used 👀